packages/react-icons/src/utils/BundledIcon.scss

```diff
@@ -0,0 +1,10 @@
+.bundled-icon-span .root-span.filled {
+    display: none;
+}
+.bundled-icon-span:hover .root-span.regular, .bundled-icon-span:active .root-span.regular .bundled-icon-span:focus .root-span.regular{
```

Comment:
`:hover` is a sensible default for toggling filled/unfilled states, but I still believe there will be lots of cases where we'd want to customize that behavior, aka making the bundled component controllable.

For example, in a command bar or left nav, those states might change based on user selection rather than hover. In ODSP we will initially only be changing color, not filled state, though that may change in time.

A better scenario would be disabling toggling if the icon is in a selected nav item. If an icon is already filled, it's probably undesirable to switch back to unfilled on hover.

Do you have concerns about adding that capability in general? It could be added to a future iteration if you'd rather proceed with current behavior for now. I just worry it'll be an adoption blocker if users need to write extra code to override default behavior.